contracts/GovNftSugar.vy

```diff
@@ -57,116 +61,125 @@ interface IToken:
   def delegates(_account: address) -> address: view
 
 # Vars
-nft: public(IGovNFT)
+factory: public(IGovNFTFactory)
+collections: public(DynArray[address, MAX_COLLECTIONS])
 
 # Methods
 
 @external
-def __init__(_nft: address):
+def __init__(_factory: address):
   """
-  @dev Set up the GovNFT contract
+  @dev Set up the GovNFTFactory contract
   """
-  self.nft = IGovNFT(_nft)
+  self.factory = IGovNFTFactory(_factory)
+  self.collections = self.factory.govNFTs()
 
 @external
 @view
-def owned(_account: address) -> DynArray[GovNft, MAX_RESULTS]:
+def all_collections() -> DynArray[address, MAX_COLLECTIONS]:
+  return self.factory.govNFTs()
+
```

Comment:
we can have an all_collections(start, end) and return the factory.govnfts(start, end). Once the number of govnft start to grow in the factory, we are going to need pagination